### ISSUE TITLE
Error box appeared while playing music

### ISSUE THREAD

**Issue by EleanorEllis** [quodlibet/quodlibet#2614]
```
An exception has occured in Quod Libet. A dump file has been saved to ~/.quodlibet/dumps/Dump_20171101_045034.txt that will help us debug the crash. Please file a new issue at https://github.com/quodlibet/quodlibet/issues/newand attach this file or include its contents. This file may contain some identifying information about you or your system, such as a list of recent files played. If this is unacceptable, send ~/.quodlibet/dumps/MiniDump_20171101_045034.txt instead with a description of what you were doing.

Quod Libet may now be unstable. Closing it and restarting is recommended. Your library will be saved.

g-io-error-quark:SWocket I/O timed out (24)
spliced line 160
   /uar/lib/pyton2.7/dist-packages/quodlibet/util/http.py
 ```

[Dump_20171101_045034.txt](https://github.com/quodlibet/quodlibet/files/1433349/Dump_20171101_045034.txt)

**Comment by declension** [comment]
Thanks for the report. 

This is now fixed in the latest builds - see #2590 for details.

**Comment by EleanorEllis** [comment]
Wow! That was a quick response!
